static/src/javascripts-legacy/projects/common/modules/experiments/tests/tailor-survey.js

```diff
@@ -13,33 +13,33 @@ define([
     'raw-loader!common/views/experiments/tailor-survey.html',
     'lib/fetch-json',
     'lodash/collections/forEach',
-    'ophan/ng'
-], function (
-    bean,
-    bonzo,
-    fastdom,
-    Promise,
-    debounce,
-    config,
-    cookies,
-    storage,
-    mediator,
-    fastdomPromise,
-    privateBrowsing,
-    quickSurvey,
-    fetchJson,
-    forEach,
-    ophan
-) {
+    'ophan/ng',
+    'lib/template'
+], function (bean,
+             bonzo,
+             fastdom,
+             Promise,
+             debounce,
+             config,
+             cookies,
+             storage,
+             mediator,
+             fastdomPromise,
+             privateBrowsing,
+             tailorSurvey,
+             fetchJson,
+             forEach,
+             ophan,
+             template) {
     return function () {
         this.id = 'TailorSurvey';
-        this.start = '2017-01-25';
-        this.expiry = '2017-03-31';
-        this.author = 'Manlio';
+        this.start = '2017-03-07';
+        this.expiry = '2017-04-30';
```

Comment:
You will also want to update the expiry in:
`common/app/conf/switches/ABTestSwitches.scala`

N.B. it is better to pick a weekday 🙃  ...otherwise a test will fail